ext/mysql2/result.h

```diff
@@ -1,5 +1,6 @@
 #ifndef MYSQL2_RESULT_H
 #define MYSQL2_RESULT_H
+#include <stdbool.h>
```

Comment:
Are there platforms these days that don't have stdbool.h available?

I'll check my Solaris and Windows environments. Off the cuff I think this will be a safe change.